common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/resetpassword/ResetPasswordChallengeApiResponse.kt

```diff
@@ -46,10 +46,10 @@ class ResetPasswordChallengeApiResponse(
     @Expose @SerializedName("challenge_channel") val challengeChannel: String?,
     @Expose @SerializedName("code_length") val codeLength: Int?,
     @Expose @SerializedName("interval") val interval: Int?,
-    @Expose @SerializedName("error") val error: String?,
+    @SerializedName("error") val error: String?,
```

Comment:
May I ask what are the criteria for distinguishing whether or not we should expose the fields?

Reply:
Whether or not the field may contain PII or other sensitive data. Nothing sensitive should be logged (the logger looks at the @Expose annotation and will only log fields that are marked as exposable).